lib/tictactoe_core/ai/minimax_ai.rb

```diff
@@ -1,6 +1,6 @@
-require 'tictactoe/marker'
+require 'tictactoe_core/marker'
 
-module TicTacToe
+module TicTacToeCore
```

Comment:
TicTacToeCore should become tic_tac_toe_core in snakecase. change in capitalisation give you an underscore.